### DISCUSSION TITLE
Unable to install

### DISCUSSION THREAD

**Discussion by tanthammar** [magic-test/magic-test-laravel#40] [Q&A]
I use Laravel 9.
When trying to install I get the following error:
```
Problem 1
    - Root composer.json requires magic-test/magic-test-laravel ^0.15.0 -> satisfiable by magic-test/magic-test-laravel[0.15].
    - magic-test/magic-test-laravel 0.15 requires illuminate/contracts ^8.0 -> found illuminate/contracts[v8.0.0, ..., 8.x-dev] but it conflicts with your root composer.json require (^9.19).
```

**Comment by Kristories** [comment]
Latest version [`v0.15`](https://packagist.org/packages/magic-test/magic-test-laravel#0.15) doesn't support Laravel 9, but you can use [`dev-master` version](https://packagist.org/packages/magic-test/magic-test-laravel#dev-master). 

Please add release for [this PR](https://github.com/magic-test/magic-test-laravel/pull/38) @mateusjatenee
